mobile-app/app/screens/AppNavigator/screens/Loans/VaultDetail/components/LoansTab.tsx

```diff
@@ -184,47 +333,238 @@ function ActionButtons({
   const navigation = useNavigation<NavigationProp<LoanParamList>>();
 
   return (
-    <View style={tailwind("mt-4 -mb-2 flex flex-row justify-between")}>
-      <View style={tailwind("flex flex-row flex-wrap flex-1")}>
-        <IconButton
-          disabled={!canUseOperations}
-          iconLabel={translate(
-            "components/VaultDetailsLoansTab",
-            "PAYBACK LOAN"
-          )}
-          style={tailwind("mr-2 mb-2 p-2")}
-          testID={`${testID}_payback_loan`}
-          onPress={() => {
-            navigation.navigate({
-              name: "PaybackLoanScreen",
-              merge: true,
-              params: {
-                vault,
-                loanTokenAmount: loanToken,
-              },
-            });
-          }}
-        />
-        <IconButton
-          disabled={!canUseOperations || vault.state === LoanVaultState.FROZEN}
-          iconLabel={translate(
-            "components/VaultDetailsLoansTab",
-            "BORROW MORE"
-          )}
-          style={tailwind("mr-2 mb-2 p-2")}
-          testID={`${testID}_borrow_more`}
-          onPress={() => {
-            navigation.navigate({
-              name: "BorrowMoreScreen",
-              merge: true,
-              params: {
-                vault,
-                loanTokenAmount: loanToken,
-              },
-            });
-          }}
-        />
-      </View>
+    <View style={tailwind("flex flex-row justify-between w-full")}>
+      <IconButton
+        disabled={!canUseOperations}
+        iconLabel={translate("components/VaultDetailsLoansTab", "PAYBACK LOAN")}
+        style={tailwind("mr-1 mb-2 p-2 w-1/2 justify-center flex-1")}
+        testID={`${testID}_payback_loan`}
+        onPress={() => {
+          navigation.navigate({
+            name: "PaybackLoanScreen",
+            merge: true,
+            params: {
+              vault,
+              loanTokenAmount: loanToken,
+            },
+          });
+        }}
+      />
+      <IconButton
+        disabled={!canUseOperations || vault.state === LoanVaultState.FROZEN}
+        iconLabel={translate("components/VaultDetailsLoansTab", "BORROW MORE")}
+        style={tailwind("ml-1 mb-2 p-2 w-1/2 justify-center flex-1")}
+        testID={`${testID}_borrow_more`}
+        onPress={() => {
+          navigation.navigate({
+            name: "BorrowMoreScreen",
+            merge: true,
+            params: {
+              vault,
+              loanTokenAmount: loanToken,
+            },
+          });
+        }}
+      />
     </View>
   );
 }
+
+const PaybackDUSD = ({
+  onCloseButtonPress,
+  paybackAmount,
+  paybackValue,
+  onSubmit,
+}: {
+  onCloseButtonPress: () => void;
+  paybackAmount: BigNumber;
+  paybackValue: BigNumber;
+  onSubmit: () => Promise<void>;
+}): React.MemoExoticComponent<() => JSX.Element> =>
+  memo(() => {
+    const { hasPendingJob, hasPendingBroadcastJob } = useSignBidAndSend();
```

Comment:
why are we using a hook from auction? maybe rename and move file to a more appropriate place if it has more use case?